README.md

```diff
@@ -34,14 +34,25 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+I would say semantic HTML makes it much easier to see what part of the code is each text.
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+The 2 big differences between "display:block" and "display:inline" is that 
+with display inline the width and height are not repected but in display block they are respected but the items dont seat side by side sometimes.
+
```

Comment:
Also, block will take up the entire width while inline will only take up the needed width.